src/view/screens/ProfileList.tsx

```diff
@@ -309,18 +285,18 @@ const Header = observer(function HeaderImpl({
         closeModal()
       },
     })
-  }, [openModal, closeModal, list])
+  }, [openModal, closeModal, list, listMuteMutation])
 
   const onUnsubscribeMute = useCallback(async () => {
     try {
-      await list.unmute()
+      await listMuteMutation.mutateAsync({uri: list.uri, mute: false})
```

Comment:
Mutes don't invalidate query, so profile list items don't show the label. Do we need that here? If you nav away and come back the muted label shows.

Reply:
Eh, probably not the biggest thing in the world